libs/shared/error/project.json

```diff
@@ -21,7 +21,7 @@
         "lintFilePatterns": ["libs/shared/error/**/*.ts"]
       }
     },
-    "test": {
+    "test-unit": {
```

Comment:
Now, to run these unit tests, instead of `nx test shared-error`, I have to use `nx test-unit shared-error`. Can we update the respective app/lib READMEs for the test command accordingly? Similarly for the `nx test-integration ...`.

```
bdanforth ~/Projects/fxa ((HEAD detached at upstream/change-test-target-for-nx-projects)) $ nx test shared-error

 >  NX   Cannot find configuration for task shared-error:test

   Pass --verbose to see the stacktrace.
```

Reply:
I'm also wondering if the next time a lib is generated via `nx`, if it will still use `"test"` instead of `"test-unit"` by default -- is that covered by the change to `nx.json`?

Reply:
I'll update the docs and look into the generate thing. I am not sure how nx decides which targets to generate. If you'd be interested in looking into this, let me know so we don't step on each others toes.

Reply:
Thanks for asking. Please go ahead! I'm interested in what you find out.